builder/parser/line_parsers.go

```diff
@@ -122,15 +125,20 @@ func parseNameVal(rest string, key string) (*Node, map[string]bool, error) {
 		if phase == inQuote {
 			if ch == quote {
 				phase = inWord
-				continue
 			}
-			if ch == '\\' {
+			// \ is special except for ' quotes - can't escape anything for '
+			if ch == '\\' && quote != '\'' {
 				if pos+1 == len(rest) {
 					phase = inWord
 					continue // just skip \ at end
 				}
+				// If next char is " then don't add the \ - its just escaping "
```

Comment:
@tiborvass why would I remove this? Its not dead code.  Note that line 135 is hit when we're in double-quotes not single.